app.js

```diff
@@ -10,10 +10,94 @@ app.locals.title = 'Test Express';
 app.use(cors());
 app.use(express.json());
 
-// All endpoints live here
+app.get('/api/v1/movies', async (req, res) => {
+  try {
+    const movies = await database('movies').select();
+    res.status(200).json(movies);
+  } catch(e) {
+    res.status(500).json({e})
+  }
+})
 
-app.get('/', (request, response) => {
-  response.send('We\'re going to test all the routes!');
-});
+app.get('/api/v1/movies/:genre', async (req, res) => {
+  try {
+    const movies = await database('movies').where('genre', 'ilike', `%${req.params.genre}%`).select();
+    if (movies.length) {
```

Comment:
I think the use of this syntax in line `24` was really tricky but I am glad you found a way around this.